<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericItemConfig</class>
 <widget class="QDialog" name="GenericItemConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Item Properties</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>441</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>431</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>Basic</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Location</string>
     </property>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="locationY"/>
       </item>
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="locationX"/>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="locationReset">
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-delete.png</normaloff>:/data/action-delete.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Stacking</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QToolButton" name="front">
         <property name="text">
          <string>To Front</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-order-front.png</normaloff>:/data/action-order-front.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="raise">
         <property name="text">
          <string>Raise</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-order-raise.png</normaloff>:/data/action-order-raise.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="lower">
         <property name="text">
          <string>Lower</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-order-lower.png</normaloff>:/data/action-order-lower.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="back">
         <property name="text">
          <string>To Back</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-order-back.png</normaloff>:/data/action-order-back.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>37</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Operations</string>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QToolButton" name="background">
         <property name="text">
          <string>To Background</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/fotowall.png</normaloff>:/data/fotowall.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="save">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-save.png</normaloff>:/data/action-save.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="del">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-delete.png</normaloff>:/data/action-delete.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Size</string>
     </property>
     <widget class="QWidget" name="layoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="sizeGridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>x</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QDoubleSpinBox" name="contentHeight"/>
       </item>
       <item row="0" column="0">
        <widget class="QDoubleSpinBox" name="contentWidth"/>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="sizeReset">
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-delete.png</normaloff>:/data/action-delete.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>401</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Transparency</string>
     </property>
     <widget class="QWidget" name="layoutWidget_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="2">
        <widget class="QToolButton" name="opacityReset">
         <property name="text">
          <string>Reset</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/data/action-delete.png</normaloff>:/data/action-delete.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="0" column="0">
        <widget class="QSlider" name="opacitySlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="outlineTab">
    <attribute name="title">
     <string>Line</string>
    </attribute>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Line Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="outlineColorPlaceholder" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Line Width</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="outlineBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QRadioButton" name="outlineNone">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No Outline</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="outlineSolid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Solid Color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="backgroundTab">
    <attribute name="title">
     <string>Background</string>
    </attribute>
    <widget class="QRadioButton" name="bgNone">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No Background</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="bgColor">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Solid Color</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="solidColorPlaceholder" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="bgGradient">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Gradient Fill</string>
     </property>
    </widget>
    <widget class="QComboBox" name="gradientSelector">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="bgImage">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture from File</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>361</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="1" column="0">
       <widget class="QLineEdit" name="imageFilenamebox"/>
      </item>
      <item row="2" column="0" rowspan="2" colspan="2">
       <widget class="QWidget" name="imagePreviewWidget" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="imageBrowseButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QRadioButton" name="bgVideo">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Video from File</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>280</y>
       <width>361</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="1" column="1">
       <widget class="QPushButton" name="videoBrowseButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" rowspan="2">
       <widget class="QWidget" name="videoPreviewWidget" native="true"/>
      </item>
      <item row="2" column="1" rowspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="videoPlayButton">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="videoFilenameBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="shadowTab">
    <attribute name="title">
     <string>Shadow</string>
    </attribute>
    <widget class="QRadioButton" name="shadowNone">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>95</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No Shadow</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="shadowSolid">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Solid Shadow</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>311</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Shadow Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="shadowColorPlaceholder" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Shadow Offset</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="shadowOffsetbox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="reflectionTab">
    <attribute name="title">
     <string>Mirror</string>
    </attribute>
    <widget class="QRadioButton" name="reflectionNone">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>No Reflection</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="reflectionEnabled">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Fancy Reflection</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Mirror Offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="mirrorOffset"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GenericItemConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GenericItemConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetLocationFields()</slot>
  <slot>resetSizeFields()</slot>
  <slot>slotToFront()</slot>
  <slot>slotToBack()</slot>
  <slot>slotRaise()</slot>
  <slot>slotLower()</slot>
 </slots>
</ui>
